plan/auto_renew_credentials.md

```
@@ -0,0 +1,118 @@
+
+    // Wait for initial prompt before sending input
+    setTimeout(() => {
+      proc.write("hi\r");
+    }, 3000);
+  });
+}
+```
+
+#### Updated `refreshCredentials()` flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)